packages/nestjs-user/src/user.types.ts

```diff
@@ -1,4 +1,10 @@
+import { PasswordStrengthEnum } from '@concepta/nestjs-password';
+
 export enum UserResource {
   'One' = 'user',
   'Many' = 'user-list',
 }
+
+export type PasswordStrengthByRoleCallback = (
```

Comment:
thinking now that it doesn't need to only work for role, can be more generic. just pass roles as one property in an object parameter